core/src/main/scala/com/databricks/labs/remorph/parsers/tsql/TSqlExpressionBuilder.scala

```diff
@@ -133,4 +327,34 @@ class TSqlExpressionBuilder
       case "<=" => ir.LesserThanOrEqual(left, right)
     }
   }
+
+  override def visitStandardFunction(ctx: StandardFunctionContext): ir.Expression = {
+    val name = ctx.funcId.getText
+    val uName = name.toUpperCase(Locale.getDefault())
+    val defnOption = functions.get(uName)
+
+    val args = ctx.expression() match {
+      case null => Seq.empty
+      case _ => ctx.expression().asScala.map(_.accept(this))
+    }
```

Comment:
```suggestion
    val args = Option(ctx.expression()).map(_.asScala.map(_.accept(this))).getOrElse(Seq.empty)
```

Reply:
OK. I have done that elsewhere as you may have seen. Was not sure which way of doing it is considered `best`